client/src/partials/journal/journal.utilities.js

```diff
@@ -255,7 +255,7 @@ angular.module('bhima.controllers')
           '  <a class="save" style="color: white; cursor: pointer;"> <span class="glyphicon glyphicon-floppy-disk saveTransaction"></span>  ' + '</a>' +
           '  <span style="padding: 5px;"></span>' + // FIXME Hacked spacing;
           '  <a class="save" style="color: white; cursor: pointer;"> <span class="glyphicon glyphicon-trash deleteTransaction"></span>  </a>' +
-          //'  <a class='deleteTransaction' style='color: white; cursor: pointer;'> <span class='glyphicon glyphicon-trash'></span>  ' + $translate('POSTING_JOURNAL.DELETE_TRANSACTION') + '</a>' +
+          '  <a class="repost" style="color: white; cursor: pointer;"> <span class="glyphicon glyphicon-refresh repost"></span> </a>' +
```

Comment:
You should put this "repost" button before the "delete" button so that it lines up with the others.  See below:

![repost](https://cloud.githubusercontent.com/assets/896472/10723741/4077a5f4-7bbe-11e5-9ab5-d7d187180300.png)